Project/Src/Gui/TextAreaDragDropHandler.cs

```diff
@@ -63,12 +63,15 @@ private static DragDropEffects GetDragDropEffect(DragEventArgs e)
 
         protected void OnDragEnter(object sender, DragEventArgs e)
         {
-            if (e.Data.GetDataPresent(typeof(string)))
+            if (IsTextDataPresent(e.Data))
                 e.Effect = GetDragDropEffect(e);
         }
 
         private void InsertString(int offset, string str)
         {
+            if (str == null)
```

Comment:
```suggestion
            if (string.IsNullOrWhiteSpace(str))
```

Reply:
@RussKie 
I think that it should be valid to drag and drop whitespace.